extensions/standard-processors/tests/unit/TailFileTests.cpp

```diff
@@ -1129,8 +1129,11 @@ TEST_CASE("TailFile finds and finishes the renamed file and continues with the n
   utils::Identifier tail_file_uuid = utils::IdGenerator::getIdGenerator()->generate();
   const core::Relationship success_relationship{"success", "everything is fine"};
 
+  // use persistent state storage that defaults to rocksDB, not volatile
+  const auto configuration = std::make_shared<minifi::Configure>();
```

Comment:
is this necessary? `TestController::createPlan` default constructs a `Configure` object if `nullptr` is provided, and by default a `RocksDbPersistableKeyValueStoreService` is constructed

Reply:
No, UnorderedMapKeyValueStoreService will be the default from now on for nullptr:
https://github.com/apache/nifi-minifi-cpp/pull/920/files/74dc246506795eb3832547da796c227d03ecb321#diff-530e3557b16fbcc2c012a6ac10efccf62258f51004a7c61d3191334cbd3e3461
In this test case, persistency is more of a requirement, so volatile is not sufficient.